rest/changes_api.go

```diff
@@ -608,10 +602,7 @@ loop:
 			break loop
 		}
 		if err != nil {
-			if h != nil {
-				h.logStatus(http.StatusOK, fmt.Sprintf("Write error: %v", err))
-			}
-			return nil, forceClose // error is probably because the client closed the connection
+			return err, forceClose
```

Comment:
Think I might need a typed error here actually in order to have the caller ignore it - having looked at the PR.

The error returned further up in the function still need to be propagated.

Reply:
Yes, it looks like we were intentionally not returning this error previously - need to review the expected handling.